src/assets/simp/build/simp.spec

```diff
@@ -23,7 +23,7 @@ Obsoletes: pupmod-simp-mcollective <= 3.0.0
 
 # Core SIMP Requirements
 Requires: pupmod-camptocamp-kmod >= 2.3.0, pupmod-camptocamp-kmod < 3.0.0
-Requires: pupmod-simp-systemd >= 2.1.0, pupmod-simp-systemd < 3.0.0
+Requires: pupmod-simp-systemd >= 2.2.0, pupmod-simp-systemd < 3.0.0
```

Comment:
pupmod-camptocamp-systemd

Reply:
should this actually just be changed to `puppet-systemd` ? Since that is what the repo is called?

Reply:
The name of the RPM will be constructed from the top-level 'name' key in the metadata.json

Reply:
ahhh gotcha, thanks!

Reply:
then should be just `camptocamp-systemd`

Reply:
No, it is constructed from name.  pupmod-<name>

Reply:
mmm, gotcha